payment/src/main/java/org/spine3/samples/lobby/payment/PaymentAggregate.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2015, TeamDev Ltd. All rights reserved.
```

Comment:
It's 2016. The header is wrong.

Copying the `.idea` files from `core-java` will do for the newly created files. But the existing files need to be updated manually.